src/components/RichContenteditable/RichContenteditable.vue

```diff
@@ -152,7 +153,7 @@ export default {
 		},
 		autoComplete: {
 			type: Function,
-			required: true,
+			default: () => {},
```

Comment:
@jancborchardt do we want autocompletion in all editors?
I tend to say yes

Reply:
Why would you enforce autocompletion in all contenteditables?

Reply:
Because that is an awesome feature and would make sense for it to be available in all areas

Reply:
It sure is awesome but that dosn't mean it's always needed.
Requiring it all the time sounds like a good recipe for scaring developers off this component IMO. It's good to offer the ability.
In the case of talk's group description, what sort of auto-completion would be desirable?

Reply:
> what sort of auto-completion would be desirable?

Depends on what you wish to write, could be a group mention, a user mention... etc etc :shrug:

Reply:
I see no need for those in that context, hence why I removed the "required" key

Reply:
Yes, I figured :)

Reply:
If it’s about autocompletion of @-mentions, I would say it would be nice to have indeed. It’s something which would also be amazing for the Notes app, Text and rich workspaces for example.
And in the case of Talk room descriptions, you could list e.g. contact people, or more info on the maintainers of an app in the team channel of an app. Makes it more interactive and friendly, and ties in nicely with the user profiles / hovercards.

So unless we have a set of specific examples where we say it would be _negative_ to allow autocompletion, then I’d say it’s good to have? I mean, it’s basically a helpful thing converting a plain text `@marco` into the proper bubble, just like a plain text `nextcloud.com` would be linkified → it’s convenient and looks nice.

Reply:
We can have different opinions bout talk's description, to me that's just noise. But regardless of that this is a basic building block and by requiring devs to pass in an autocompletion function we're making it more tedious and less versatile and thus less likely to be adopted. 
... Anyway, reverting..

Reply:
@ma12-co I guess you already have the whole autocompletion thing already/almost available on talk.
Feel free to wrap it into a component in the talk repo for you to use (especially for the [`userData`](https://github.com/nextcloud/nextcloud-vue/blob/7372d9a62bffbc3f7aea3c27a4f62cbcf8c2d5b0/src/mixins/richEditor/index.js#L37) needed to render the initial mentions)

Reply:
> to me that's just noise

@ma12-co that's fine, and in that case you wouldn't use it.
But other systems with @-mention functionality do usually have that everywhere, not only in their chat component, e.g. Dropbox in their "Rich workspace" equivalent, or Github in their wikis, Trello in their comments, etc etc.